### ISSUE TITLE
Usersnap Feedback - I'm adding a case that is related to funding youth programs and participat[...]

### ISSUE THREAD

**Issue by ascott** [participedia/usersnaps#886]
**Sender**: alanna.scott@gmail.com
**Comment**: I'm adding a case that is related to funding youth programs and participatory budgeting. I thought to add a Youth issue in the General Issues field, but we don't have a youth related issue. Is there another issue i should use in it's place?
[Open #582 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-net-9cb6d041/582)

<a href='https://usersnappublic.s3.amazonaws.com/2019-10-06/00-49/0861fc7e-339d-49df-89fe-b0c81e3a12f5.png'>![Usersnap Feedback - I'm adding a case that is related to funding youth programs and participat(...)](https://usersnappublic.s3.amazonaws.com/2019-10-06/00-49/fcd78a03-4e94-42d8-821f-bdfccbea2db2.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2019-10-06/00-49/0861fc7e-339d-49df-89fe-b0c81e3a12f5.png'>Download original image</a>

**Browser**: Chrome 77 (macOS High Sierra)
**Referer**: [https://participedia.net/case/new](https://participedia.net/case/new)
**Screen size**: 1680 x 1050 **Browser size**: 1660 x 928

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by plscully** [comment]
@ascott I have a few ideas about to handle this. I will check with @scottofletcher first, though, because this may have implications for how we are categorizing other participatory budgeting cases

**Comment by scottofletcher** [comment]
hi guys, I made a usersnap a while ago about this kind of problem (#850). there are numerous cases like this (mostly 'parents') that have more than one gen issue/specific topic and it could change from year to year (eg. a PB initiative organized around a different theme each cycle - as in the case of Victoria where this year projects had to be focused on youth, last year they just had to "improv[e] life in Victoria"). 

Here's my responses to the email Pat sent me on this: 

- What is the most appropriate default ‘General Issue” for PB when it is a wide-open process that doesn’t define one or more areas of interest?  ‘Planning and Development’ and ‘Economics’ come to mind, but neither seems exactly right. If we decide that we need to add a new General Issue, what should it be? Budgeting; Budgetary Governance; …?
--> This is what I asked in my original usersnap. I think Planning & Development makes the most sense except in cases of PB that are on budget cutting, in which case Economics might work better. 

- I did a quick read of the Victoria page on PB and it was hard to get a handle on how they define their overall goals, although the 2019 round does appear to have focused on youth issues.
--> see my comment above^ - this year the theme was youth but next year might be something different. best to put Planning & Development for the parent and create a component for the 2019 cycle and select 'Youth' under specific topics. again, we may need to rethink how to categorize 'parent' cases like these that (theoretically) have an indefinite amount of issues/topics. I hesitate to leave the sections blank, but perhaps that's best. I think the most important fields to fill out for parents are purpose, approach, and gen + specific types of methods/tools/techniques.  

- What’s your sense of how to document PB programs that continue year after year? Should we create a master for a city’s overall program, linked to components for each year? Assuming we figure out how to download the resulting data properly, this would allow users to see longer-term development of the PB work.
--> yes, that's how I've been doing it and advising other contributors to do. Angus used that structure for Winnipeg and I think it worked well. we also now have a couple cycle-specific components for Madrid's PB.

- Specific Topics – I know you are tracking these so that we can later review them and decide what the next iteration should be. In the meantime, do you agree that the best ones for Victoria’s 2019 PB are ‘Budget-Local’ and ‘Youth Issues’?
--> Yes, actually I suggested condensing the three Budget topics into one (since they don't cover all of the geographic scope selections we have, and filtering by scope and/or type of organizing or funding entity is more accurate anyways

**Comment by plscully** [comment]
Thank you, @scottofletcher . I agree with everything you suggested above, except using 'Planning & Development' as the default General Issue when a parent case has components with more than one General Issue. P&D has a specific meaning related to how jurisdictions implement things such as land-use zoning, transportation planning, etc. one example is [this stub](https://participedia.net/case/5241) I created a while ago. And here's an example of how a municipal department that focuses on these types of issues [https://detroitmi.gov/departments/planning-and-development-department](url). ... 
When working with a parent case has components with more than one General Issue, we could instead either (a) follow Jesi's advice and leave the GI field blank, or (b) create a new GI value of "Miscellaneous"

**Comment by scottofletcher** [comment]
yes, sorry I didn't mean that we should always use P&D for parent cases, but it does seem to make the most sense for parent _participatory budgeting_ cases. from the cases I've seen, most source/receive project submission that entail some type of construction/development (eg. new parks and rec facilities seem to be a popular submission/winner). I'm not sure 'Miscellaneous' captures it - that would imply that none of the general issues apply when, in fact, it's the opposite: too many of them apply. I would suggest 'Multiple' or 'Numerous' instead

**Comment by scottofletcher** [comment]
also, we have to remember that each user interprets these fields differently. for example, I've noticed that users tend to select 'Information and Communications Technology' on cases that involve some form of online participation. technically those cases aren't being held on the issue of ICT, they just use it as an interface to discuss, say, planning and development.

**Comment by plscully** [comment]
"Multiple" instead of miscellaneous works for me.

**Comment by ascott** [comment]
thanks for all this
context pat and scott. 

adding a ‘Multiple’ value to the general issues field doesn’t seem valuable to me from a user perspective or a research perspective. what does an entry being marked as Multiple 
tell us?

i’m also wondering about the rationale for separating general issues from
specific issues. i added this entry as an exercise after reading Jesi’s most recent blog post and so coming to the quick submit form and only being able to select a general issue, it didn’t occur to me to look for a specific issue field to add the Youth value. Could it be beneficial to combine these two fields so inexperienced contributors can always find the issues they are looking for, without relying on the insider knowledge that we are categorizing an entry's issues at two levels? (specific & general) Alternatively could we move the specific issues field right below general issues to make it more clear?

**Comment by scottofletcher** [comment]
@ascott I think 'Multiple' does have some value, as it would indicate that an initiative used a potentially 'deeper' form of engagement (eg. participant set the agenda and have a say in the outcomes) or it is a more 'institutionalized' example of DI (ie. it wasn't just a 'one-off', it was a more sustained effort to include citizens in the policy-making process on different issues or on different levels)

RE gen issues/specific topics: I don't have strong feelings on this, but maybe check out [my edited list](https://docs.google.com/spreadsheets/d/1Kj0EtjeW9OyKn6cU1lq-XDq2-mbP9eIA6ey7DKaTkjo/edit#gid=0) of specific topics. I wouldn't be opposed to cutting it down even further and getting rid of gen issues altogether. however, as it stands, the specific issues drop-down is far too unwieldy to include in quick submit (IMO).

**Comment by ascott** [comment]
@scottofletcher i see what you are saying about Multiple, but it's a very vague term, and open to many possible interpretations. 

My suggestion would be to merge general and specific values into one field, and allow the user to choose up to 3 or 5 issues for the entry. Since we have the new autocomplete select, including a long list of options in quick submit shouldn't be a problem since users can quickly filter the list to find what they are looking for.

**Comment by scottofletcher** [comment]
ok yes I agree that it can be interpreted various ways and would be in favour of combining the two fields. however, I still think we should cut down the final list to at least the one I suggested (which is still really long)

**Comment by plscully** [comment]
I agree that we should create a single list and maybe label it 'Issues/Topics' or something similar that is clear and short. @scottofletcher and I can work on the list. If @ascott and @jesicarson can give us a deadline for when you need the new list, we will meet it.

**Comment by plscully** [comment]
One more thing, is there any utility in including an 'other' value to the list, even if we don't ask people to tell us what they see missing? I don't have strong feelings about this, just curious.

**Comment by jesicarson** [comment]
We removed other on purpose, as it has no meaning. As you mentioned, I tend to just leave fields blank if there’s no matching data value, and write about it in the narrative if necessary. 
Moving specific issues to the quick sub form could be ok, although we are aiming for as low barrier as possible (least amount of fields). You can say that the case is youth related in the short description too.

**Comment by jesicarson** [comment]
Or combine the fields into 1, sure. That works!

**Comment by jesicarson** [comment]
Regarding deadline, this is a data model change so perhaps it should be considered with the other changes and we can schedule a data model review as discussed. I don’t have a date in mind for that, but possibly next June? 1 year from launch? Thoughts?

**Comment by plscully** [comment]
I don't have strong feelings about the deadline, although I agree that it needs to be included in the data model review.

**Comment by ascott** [comment]
a quick win here would be to place the Specific Topics field right below the General Issues field in the quick submit, so at least a user can see all of the fields related to issues in one place. This wouldn't require any backend or model changes and would be like a 5 minute front-end change.

**Comment by plscully** [comment]
Works for me!

**Comment by jesicarson** [comment]
I’m fine with that.
